content/news/028/index.md

```diff
@@ -135,6 +135,26 @@ They also worked on a [tutorial][bitgun-tutorial].
 [bitgun-missions]: https://twitter.com/LogLogGames/status/1464009563976392713?s=20
 [bitgun-tutorial]: https://twitter.com/LogLogGames/status/1461898845810348033?s=20
 
+### [Halloween Mahjong Solitaire][mahjong-github]
+
+![Halloween Mahjong Solitaire screenshot](https://user-images.githubusercontent.com/61752540/144722844-de0511a0-0f7b-4006-9d64-680b6406d984.png)
+_Screenshot of Halloween Mahjong Solitaire_
```

Comment:
Please make sure to extract all links to the bottom :)

Reply:
I tried to upload the file directly to: https://github.com/rust-gamedev/rust-gamedev.github.io/tree/source/content/news/028/

but I don't have write access. Is this something that you can do so that the image link is to an image released with the newsletter?

Reply:
That URL is for our version of the repo, you'll need to go to your fork if you want to upload the file through the web UI: https://github.com/Syn-Nine/rust-gamedev.github.io/tree/patch-1/content/news/028

Reply:
Ah, we actually just upload the file to the same directory then make a relative reference to it. You should be able to add the file on your branch, it seems like you're trying to add it right to source? If you need, I can handle the change :)

Reply:
Here's an example: `![The Raytracer Challenge](the-raytracer-challenge.jpg)` where that jpg is in the same directory.

Reply:
Ah, ok, I uploaded the file to my branch and updated the link. Thanks!